OpenMarket/OpenMarket/Error/Error.swift

```diff
@@ -9,11 +9,14 @@ import Foundation
 
 enum OpenMarketError: Error, LocalizedError {
     case badStatus(file: StaticString, line: UInt)
+    case invalidURL(file: StaticString, line: UInt)
```

Comment:
```suggestion
    case badStatus(file: StaticString = #file, line: UInt = #line)
    case invalidURL(file: StaticString = #file, line: UInt = #line)
```
이렇게 정의해주면 사용성이 조금 더 좋아질것 같아요

Reply:
[43682a9](https://github.com/yagom-academy/ios-open-market/pull/218/commits/43682a91bf6275efd3c7f7daebb970d1a65fa61c) 에서 수정했습니다. 사용하는 곳에서 값이 결정되니까 저렇게 사용하는 게 좋네요. 좋은 팁 감사합니다.